src/lib/components/lines-and-dots/event-type-filter.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+  import Icon from '$lib/holocene/icon/icon.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import {
+    allEventTypeOptions,
+    compactEventTypeOptions,
+  } from '$lib/models/event-history/get-event-categorization';
+  import { eventTypeFilter } from '$lib/stores/filters';
+  import { temporalVersion } from '$lib/stores/versions';
+  import { isVersionNewer } from '$lib/utilities/version-check';
+
+  import { CategoryIcon } from './constants';
+
+  export let compact = false;
+
+  $: options = (compact ? compactEventTypeOptions : allEventTypeOptions).map(
+    (o) => ({
+      ...o,
+      label: translate(o.label),
+      icon: CategoryIcon[o.value],
+    }),
+  );
+
+  $: {
+    if (isVersionNewer('1.21', $temporalVersion)) {
+      options = options.filter(({ value }) => value !== 'update');
+    }
+  }
+
+  const onOptionClick = ({ value }) => {
```

Comment:
Seeing the following error when unselecting an event type with the event details open (in all three views) ⬇️ 
  
<img width="1067" alt="Screenshot 2024-04-04 at 5 02 46 PM" src="https://github.com/temporalio/ui/assets/15069288/eb851d67-2992-4ffa-845c-d08c8184fd67">

It appears to leave the page in a broken state.

Reply:
Good catch, I now clear the active events when you pick a filter